core/hw/aica/aica_if.cpp

```diff
@@ -18,6 +18,7 @@ u32 VREG;//video reg =P
 u32 ARMRST;//arm reset reg
 u32 rtc_EN=0;
 int dma_sched_id;
+u32 RealTimeClock;
```

Comment:
In the future I think we should organize this thing in a struct called dc or such. Will make future refactoring easier